benchmark_app.go

```diff
@@ -66,6 +70,10 @@ func (c BenchmarkConfig) Validate() error {
 		return errors.New("must specify log filename")
 	}
 
+	if c.DatabaseConfig.ConnectionMultiplier != 1 && c.RateControlConfig.Concurrency == 0 {
+		return errors.New("must specify -concurrency if -connectionmultiplier is specified")
+	}
+
```

Comment:
I decided that it still made sense to have the user explicitly state what they're multiplying, rather than having the multiplier operating on an implicit `concurrency` value.